aiochclient/client.py

```diff
@@ -146,7 +159,7 @@ def __init__(
                     async for line in resp.content:
                         yield rf.new(line)
 
-    async def execute(self, query: str, *args, json: bool = False) -> None:
+    async def execute(self, query: str, *args, json: bool = False, params=None) -> None:
```

Comment:
Can you update documentation string with some examples how to use params argument?

Reply:
Done